src/pyvesync/vesyncbasedevice.py

```diff
@@ -42,3 +45,10 @@ def configure(self, details):
         self.speed = details.get('speed', None)
         self.extension = details.get('extension', None)
         self.current_firm_version = details.get('currentFirmVersion', None)
+
+        if self.device_type is not None:
```

Comment:
what does this do? function functionality?

Reply:
This is the part of a test I made to ensure the device is configured correctly but I figured it would be redundant if the devtype in the factory already determines the device class. Consider it gone